sr_remote_warehouse/CMakeLists.txt

```diff
@@ -0,0 +1,188 @@
+cmake_minimum_required(VERSION 2.8.3)
```

Comment:
That's a very good idea, but put it in sr_config since it is something that could be modified by the end user.